audio-worklet/index.html

```diff
@@ -36,33 +34,11 @@ <h1>
           AudioWorklet
         </h1>
         <p><a href="https://webaudio.github.io/web-audio-api/#AudioWorklet"
-          target="_blank">AudioWorklet</a> is currently available on
-          <strong>Chrome 66 (or later)</strong> by default.
-        </p>
-        <p>
-          For Chrome 64 and 65, AudioWorklet is available via <a href=
-          "https://docs.google.com/forms/d/e/1FAIpQLSfO0_ptFl8r8G0UFhT0xhV17eabG-erUWBDiKSRDTqEZ_9ULQ/viewform">
-          Origin Trial</a> or behind the experimental flag. Using Origin Trial
-          is highly recommended for AudioWorklet so that the page can be served
-          to the actual visitors. For the local development, use the following
-          <a href=
-          "https://www.chromium.org/developers/how-tos/run-chromium-with-flags">
-          command line option</a>.
-        </p>
-        <pre><code>--enable-blink-features=AudioWorklet</code></pre>
-        <p>
-          Although it is <strong>NOT</strong> recommended, you can enable
-          Experimental Web Platform Features in Chrome's flag setting. Note that
-          this setting enables all the experimental features in browser. Copy
-          and paste the following URL in Chrome, enable the feature and relaunch
-          the browser.
-        </p>
-        <pre><code>chrome://flags/#enable-experimental-web-platform-features</code></pre>
-        <p>
-          If the status indicator on the top-right corner of this page is not
-          green, it means either your browser does not support AudioWorklet
-          or you need to follow the instruction above to activate the
-          feature.
+          target="_blank">AudioWorklet</a> is available on <strong>Chrome 66 (or
+          later) by default</strong>. If the status indicator on the top-right
+          corner of this page is not green, it means either your browser does
+          not support AudioWorklet or you need to update Chrome browser to
```

Comment:
I'd rephrase this slightly:

it means your browser does not support currently AudioWorklet.  If you are using Chrome, please update to Chrome 66.